### PR TITLE
disambiguate the entrypoint launch sequence

### PR DESCRIPTION
**Pull request by sameersbn**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM gcr.io/stacksmith-images/minideb:jessie-r8
+FROM gcr.io/stacksmith-images/minideb:jessie-r9
```
Comment (sameersbn) [inline Dockerfile]: changeset depends on `gcr.io/stacksmith-images/minideb:jessie-r9`

File: rootfs/app-entrypoint.sh [hunk 2]
```diff
@@ -21,11 +14,9 @@ export MARIADB_USER=${MARIADB_USER:-"root"}
 export MARIADB_HOST=${MARIADB_HOST:-"mariadb"}
 export MARIADB_PORT=${MARIADB_PORT:-"3306"}
```
Comment (sameersbn) [inline rootfs/app-entrypoint.sh]: should we move these to the Dockerfile?

File: rootfs/app-entrypoint.sh [hunk 3]
```diff
@@ -21,11 +14,9 @@ export MARIADB_USER=${MARIADB_USER:-"root"}
...
 
-if [[ "$1" == "nami" && "$2" == "start" ]] ||  [[ "$1" == "/init.sh" ]]; then
-   for module in redmine; do
-    initialize $module
-   done
-   echo "Starting application ..."
+if [[ "$1" == "nami" && "$2" == "start" ]] || [[ "$1" == "/init.sh" ]]; then
+  nami_initialize redmine
```
Comment (sameersbn) [inline rootfs/app-entrypoint.sh]: `nami_initialize` is a helper function made available by the `/opt/bitnami/base/helpers` import. it allows for initialising one or more nami modules f.e `nami_initialize apache php drupal`. For each listed module the function will initialize the module if it is not in the initialized module state.

Comment (sameersbn) [inline rootfs/app-entrypoint.sh]: it also handles specifying the `module-inputs.json` file it it exists.

File: rootfs/app-entrypoint.sh (hunk 2 above)
Comment (prydonius) [inline rootfs/app-entrypoint.sh]: Yeah, I think defaults should be in the Dockerfile

**Pull request merged**